darwin/utils/utils.py

```diff
@@ -993,13 +994,17 @@ def split_video_annotation(annotation: dt.AnnotationFile) -> List[dt.AnnotationF
         no ``frame_url`` attribute.
     """
     if not annotation.is_video:
-        raise AttributeError("this is not a video annotation")
-
-    if not annotation.frame_urls:
-        raise AttributeError("This Annotation has no frame urls")
-
+        raise AttributeError("This is not a video annotation")
+
+    if not annotation.frame_count and not annotation.frame_urls:
+        raise AttributeError("This Annotation has no frames")
+    urls = (
+        annotation.frame_urls
+        if annotation.frame_urls
+        else [None] * (annotation.frame_count or 1)
+    )
     frame_annotations = []
-    for i, frame_url in enumerate(annotation.frame_urls):
+    for i, frame_url in enumerate(urls):
```

Comment:
In the case of a long video frame_url get's passed into the AnnotationFile.image_url field as None. Perhaps this is not the correct assumption but I couldn't reason about what an image_url would be in this case, and none of our tests fail.